dpnp/dpnp_utils/dpnp_utils_linearalgebra.py

```diff
@@ -2063,7 +2064,9 @@ def dpnp_matmul(
             )
             host_tasks_list.append(ht_blas_ev)
             if not row_major:
-                result = dpnp.reshape(result.ravel(), result.shape, order="F")
+                result = dpnp.ascontiguousarray(
+                    dpnp.reshape(result.ravel(), result.shape, order="F")
+                )
```

Comment:
Isn't that the same as `result = dpnp.asfortranarray(result)`

Reply:
It is not the same as `result = dpnp.asfortranarray(result)` but some improvement may be possible here as mentioned in the other [comment ](https://github.com/IntelPython/dpnp/pull/1793#discussion_r1590894812)which will be investigated in future  PRs.